app/utils.py

```diff
@@ -74,6 +75,39 @@ def sanitize_email(email_address: str, not_lower=False) -> str:
     return email_address
 
 
+class NextUrlSanitizer:
+    def __init__(self, allowed_domains: List[str]):
+        self.allowed_domains = allowed_domains
+
+    def sanitize(self, url: Optional[str]) -> Optional[str]:
```

Comment:
If we want to use class to organise code, I think we can use `@staticmethod`, this way we don't have to instantiate a new instance of `NextUrlSanitizer`

Reply:
`NextUrlSanitizer` converted to a class with static methods :+1: